jupyter_vscode_proxy/__init__.py

```diff
@@ -61,14 +61,16 @@ def _get_cmd(port: int) -> List[str]:
 
 def setup_vscode() -> Dict[str, Any]:
     executable = os.environ.get("CODE_EXECUTABLE", "code-server")
+    use_code_server_icon = os.environ.get("USE_CODE_SERVER_ICON", "false").lower()
+    icon = "code-server.svg" if use_code_server_icon == "true" else "vscode.svg"
```

Comment:
> Shouldn't we always use this icon?

When the executable is using `code-server` yes, but when it is `openvscode-server`... well, it is not the code-server. I wanted avoid breaking the experience for existing users, but I was also really tempted to rewrite it to:

```suggestion
    icon = "code-server.svg" if executable == "code-server" else "vscode.svg"
```

What do you think @xhochy? Feel free to commit my suggestion directly :)